apps/src/sites/code.org/pages/public/learn/local.js

```diff
@@ -117,31 +127,189 @@ function loadMap(locations) {
 
   // Reset the map.
   $('#gmap').html('');
-  gmap = new Maplace();
-
-  var map_options = {
-    map_options: {
-      set_center: [lat, lng],
-      zoom: 12
-    },
-    controls_type: 'list',
-    controls_on_map: false
-  };
+  if (window.location.search.includes('mapbox')) {
+    const featureList = locations.map((location, index) => ({
+      type: 'Feature',
+      properties: {
+        description: location.html,
+        title: location.title,
+        index
+      },
+      geometry: {
+        type: 'Point',
+        coordinates: [parseFloat(location.lon), parseFloat(location.lat)]
+      }
+    }));
+    gmap = new mapboxgl.Map({
+      container: 'gmap',
+      style: 'mapbox://styles/mapbox/streets-v11',
+      zoom: 10,
+      minZoom: 1,
+      center: [lng, lat]
+    });
+    gmap.on('load', function() {
+      gmap.loadImage(
+        'https://docs.mapbox.com/mapbox-gl-js/assets/custom_marker.png',
```

Comment:
Maybe we should make our own copy of this image rather than relying on their docs server?

Reply:
Probably, for whatever image we end up using. This marker is a placeholder, I'm not sure what we're actually going to use.

Reply:
I agree, though I don't think this blocks this PR